src/config/corsConfig.ts

```diff
@@ -0,0 +1,18 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+const corsConfig = () => {
+
+  const isProduction = process.env.NODE_ENV === 'production';
+
+  const corsOptions = {
+    origin: isProduction ? 'TBD' : 'http://localhost:3000', //frontend
```

Comment:
Needed these new corsOptions in order to be able to get HTTP-only cookies and access them from the frontend.